### PR TITLE
chore: Update moby version to 19.03.12

### PR DESCRIPTION
**Pull request by bowang-666**

### CODE REVIEW

**Comment by mboersma** [comment]
@bowang-666 could you `git commit --amend` and force push this so the commit message is [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) style? Something like you have as the PR title:

```
chore: update moby version to 19.03.11
```

File: pkg/api/const.go [hunk 1]
```diff
@@ -457,7 +457,7 @@ const (
 	// DefaultKubernetesDNSServiceIPv6 specifies the IPv6 address that kube-dns listens on by default. must by in the default Service CIDR range.
 	DefaultKubernetesDNSServiceIPv6 = "fd00::10"
 	// DefaultMobyVersion specifies the default Azure build version of Moby to install.
-	DefaultMobyVersion = "3.0.10"
+	DefaultMobyVersion = "19.03.11"
```
Comment (cpuguy83) [inline pkg/api/const.go:460]: 19.03.12 is latest.

Can't believe we missed this one.

Comment (bowang-666) [reply pkg/api/const.go:460]: seems on github it says latest release is 19.03.11: https://github.com/Azure/moby/releases
also 11 was released on Jun.26 but 12 was released on Jun.25 which looks weird

Comment (cpuguy83) [reply pkg/api/const.go:460]: That's just tag date vs when the release was created in GH.

Comment (cpuguy83) [reply pkg/api/const.go:460]: One is automatic when a tag is created, the other is manual after the fact.

Comment (bowang-666) [reply pkg/api/const.go:460]: OK sounds good. I'll update to .12

**Pull request merged**